src/pubtools/sign/signers/msgsigner.py

```diff
@@ -375,12 +381,9 @@ def container_sign(self: MsgSigner, operation: ContainerSignOperation):
             messages.append(message)
 
         signer_results = MsgSignerResults(status="ok", error_message="")
-        signing_key = operation.signing_key
-        if signing_key in self.key_aliases:
-            signing_key = self.key_aliases[signing_key]
-            LOG.info(f"Using signing key alias {signing_key} for {operation.signing_key}")
+        LOG.info(f"Using signing key alias {signing_key} for {operation.signing_key}")
```

Comment:
Could you update it to log only when aliases exist?

Reply:
yep